gensim/scripts/segment_wiki.py

```diff
@@ -54,7 +51,7 @@
 logger = logging.getLogger(__name__)
 
 
-def segment_all_articles(file_path, min_article_character=200):
+def segment_all_articles(file_path, min_article_character=200, workers=None):
```

Comment:
Why `workers=None` by default? If default is `number of cores - 1`, this should be `workers=multiprocessing.cpu_count() - 1` or something equal (same for all `workers=...`)

Reply:
It is (see the docs).

Reply:
Why not substitute the real default value (instead of `None), when you have it? Why needed `None` here?

Reply:
I find it more explicit -- a person viewing the signature would see a number, and perhaps not realize this number is "variable" (will be different on a different system).

In other words, I find it a little "less surprising" when only constants are in default parameters. Anything dynamic/mutable is handled by `None` instead.